src/main/java/com/school/project/model/entities/Group.java

```diff
@@ -1,13 +1,33 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
 import java.util.Date;
 import java.util.List;
 
+@Entity
+@Table(name = "`GROUP`")
```

Comment:
should work without ` also, pls try

Reply:
I tried without `` ``, when starting it gives an error syntax MySQL